### DISCUSSION TITLE
admonitions inside code annotations?

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#3903] [Q&A] [answered]
Hi, I am running 4.15.0 insiders and wondering if it is possible to use
```
!!!note
```
admonition inside code annotation after some regular text without resorting to html?

the example I am trying to make to work:

<pre>
```yaml
test:
  val: 2 # (1)!
```

1. some text  
    !!!note
        test note
</pre>

**Comment by UltraInstinct05** [comment, marked as answer]
Seems to work just fine for me on the latest version of `mkdocs-material` public version. Just use admonitions like you normally would inside code annotation numbers.

![image](https://user-images.githubusercontent.com/34304644/167394651-1c4bf947-480d-42ed-92ad-becddc33f2ea.png)

**Reply by hellt** [reply]
My bad, I was actually asking about an admonition being not the first element in the annotation.

Consider the following example which I am struggling to make to work

<pre>
```yaml
test:
  val: 2 # (1)!
```

1. some text  
    !!!note
        test note
</pre>

The idea is to place an admonition after some text in the annotation

**Reply by UltraInstinct05** [reply]
You just need to add a newline after `some text`

![image](https://user-images.githubusercontent.com/34304644/167398190-9b7aaa9f-9fbe-400f-aaf4-c810b396d780.png)

**Reply by hellt** [reply]
thanks @Ultra-Instinct-05 !
adding newlines before and after the admonition made it all work nicely

<img width="220" alt="image" src="https://user-images.githubusercontent.com/5679861/167402599-33fbda5a-7817-4f3e-a044-4ad3bdf73083.png">

<img width="606" alt="image" src="https://user-images.githubusercontent.com/5679861/167402663-904ec794-5ec5-4beb-ba1d-350fa577c4cb.png">
